php/alsearch.php

```diff
@@ -133,7 +136,11 @@
         ];
     } else {
         // Editing an existing airline
-        $sql = "UPDATE airlines SET name=?, alias=?, country=?, iata=?, icao=?, callsign=?, mode=?, active=? WHERE alid=? AND (uid=? OR ? IN (?))";
+        $sql = <<<SQL
```

Comment:
I guess we probably don't need to use HEREDOC - https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc